src/radical/entk/appman/appmanager.py

```diff
@@ -138,17 +140,21 @@ def __init__(self,
     #
     def _read_config(self, config_path, hostname, port, username, password,
                      reattempts, resubmit_failed, autoterminate,
-                     write_workflow, rts, rmq_cleanup, rts_config):
+                     write_workflow, rts, rmq_cleanup, rts_config, config_params):
 
         if not config_path:
             config_path = os.path.dirname(os.path.abspath(__file__))
 
         config = ru.read_json(os.path.join(config_path, 'config.json'))
 
+        # config_params (2nd dictionary) overwrites values if duplicated
+        config = { **config, **config_params }
```

Comment:
Maybe `config.update(config_params)` would be more explicit and readable?

Reply:
I agree, thanks for the comment.